pycypher/local_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e -x
+cd "$(dirname -- "$0")"
+
+export LD_LIBRARY_PATH=../src/lib/.libs/:$LD_LIBRARY_PATH
+export LIBRARY_PATH=../src/lib/.libs/:$LIBRARY_PATH
+export C_INCLUDE_PATH=../src/lib/:$C_INCLUDE_PATH
+
+./generate.py
+./setup.py build_ext -i
```

Comment:
Why separate scripts for setup and generate?

Reply:
Every Python package needs to have a setup.py in it's root, it's a standard. generate.py is separate because it depends on files from parent directory and setup.py is used by Python packaging tools so it's better it does not make assumptions on it's location in the filesystem.

Reply:
Ok. Perhaps generate.py should then be run automatically by configure, or otherwise included into it?

Reply:
This sounds good

Reply:
Sounds good, I will look into that.

Reply:
After a bit of thinking I concluded that pycypher will not be included in `make dist` so I think this no longer apply. The `docker-pycypher` target invokes it, and so the `make check` as part of testing script. `generate.py` runs instantly so it is not a problem that it will be re-run each time `make check` is done.